src/WebJobs.Script.WebHost/Management/WebFunctionsManager.cs

```diff
@@ -214,7 +214,14 @@ private async Task<(bool, string)> InternalSyncTriggers(IEnumerable<JObject> tri
             var content = JsonConvert.SerializeObject(triggers);
             var token = SimpleWebTokenHelper.CreateToken(DateTime.UtcNow.AddMinutes(5));
 
-            var url = $"https://{Environment.GetEnvironmentVariable(EnvironmentSettingNames.AzureWebsiteHostName)}/operations/settriggers";
+            var protocol = "https";
```

Comment:
There's already tests for SyncTriggers [here](https://github.com/Azure/azure-functions-host/blob/ef311c8c4a46e6494e0b6b4af359586bb1295667/test/WebJobs.Script.Tests/Managment/WebFunctionsManagerTests.cs). Can you factor out the request builder stuff so it is unit testable (e.g. `internal static HttpRequestMessage BuildSyncTriggersRequest(IEnumerable<JObject> triggers)` ) and add a test that verifies your new switching logic? Have it validate the other request properties/headers as well.